sycl/source/detail/scheduler/scheduler.hpp

```diff
@@ -432,14 +432,14 @@ class Scheduler {
 
   /// Allocate buffers in the pool for a provided stream
   ///
-  /// \param Pointer to the stream object
-  /// \param Size of the stream buffer
-  /// \param Size of the flush buffer for a single work item
```

Comment:
Will it really help without naming of arguments?

Reply:
I believe it's better to leave the documentation with named arguments as it was planned by the author